### PR TITLE
Complete the renaming of /profile to /accounts in routes, layouts, and templates

### PR DESCRIPTION
**Pull request by finitud**
TO BE MERGED AFTER #3593 !

This follows from #3593 and finalizes the first item in https://trello.com/c/G3kr1awd/392-account-settings-profile-etc-forms-updates

### CODE REVIEW

**Comment by nickstenning** [comment]
Needs a rebase -- looks like this is based of a branch that's a couple of days old.

File: h/api/views.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def read(annotation, request):
 def read_jsonld(annotation, request):
     request.response.content_type = 'application/ld+json'
     request.response.content_type_params = {
-        'profile': AnnotationJSONLDPresenter.CONTEXT_URL}
+        'account': AnnotationJSONLDPresenter.CONTEXT_URL}
```
Comment (nickstenning) [inline h/api/views.py]: Oops, misfire :)

This one should remain 'profile'.

**Comment by finitud** [comment]
Rebased and ready for review/merge!

**Comment by nickstenning** [comment]
LGTM.

**Pull request merged**